hack/build-manifests.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+# TODO: If we create more hack scripts this should go in common
+# and be sourced
+PROJECT_ROOT="$(
+	cd "$(dirname "$BASH_SOURCE[0]")/../"
+	pwd
+)"
+
+# TODO: Move this to deploy
+DEPLOY_DIR="${PROJECT_ROOT}/deploy"
+STD_DEPLOY_DIR="${DEPLOY_DIR}/standard"
+CONVERGED_DEPLOY_DIR="${DEPLOY_DIR}/converged"
+CSV_VERSION="0.0.1"
+
+(cd ${PROJECT_ROOT}/tools/manifest-templator/ && go build)
+
+templates=$(cd ${PROJECT_ROOT}/templates && find . -type f -name "*.yaml.in")
```

Comment:
```find ${PROJECT_ROOT}/templates -type f -name "*.yaml.in"```

Reply:
I left the `cd ... & find` to nicely get the relative paths from the `templates/` directory. That explains why it's in KubeVirt's scripts.